src/config/constants.js

```diff
@@ -108,3 +108,4 @@ export const UPDATE_CHECK_FEED = stringFromEnv(
 export const MAX_ACCOUNT_NAME_SIZE = 50;
 
 export const MOCK_DATA_SEED = floatFromEnv("MOCK_DATA_SEED", Math.random());
+export const MODAL_OPERATION_DETAILS = "MODAL_OPERATION_DETAILS";
```

Comment:
please inline all the MODAL_* string into `"MODAL_OPERATION_DETAILS"` (which ultimately we should not name it like this but just "operationDetails", we need to think of it like a route)